src/api/app/views/layouts/webui/responsive_ux/_logged_navigation.html.haml

```diff
@@ -0,0 +1,27 @@
+.navbar-collapse.offcanvas-collapse.navbar-dark
+  .navbar-nav
+    .nav.justify-content-end.py-2
+      %button.navbar-toggler{ 'type': 'button', data: { toggle: 'offcanvas' }, aria: { expanded: 'false', label: 'Toggle navigation' } }
+        %i.fas.fa-times
+  %ul.navbar-nav.ml-auto.pt-0.text-nowrap
+    %li.nav-item
+      = link_to(user_path(User.session!), id: 'link-to-user-home', class: 'nav-link') do
+        = User.session!.login
+    %li.nav-item
```

Comment:
Why repeat tasks? I have them at the bottom.

Reply:
We dropped "Tasks" link from the menu. Now it only appears on the bottom navigation bar for small viewports and clearly visible on the header in large viewports.

![Screenshot_2020-02-04 Open Build Service](https://user-images.githubusercontent.com/2581944/73751186-df797b80-475e-11ea-88ff-6f48adf665db.png)